assets/script.js

```diff
@@ -62,17 +63,20 @@ const getArchiveCards = i => {
 getArchiveCards(current)
 
 // night mode feature
-$('#toggle-box-checkbox').on('change', function() {
+document.getElementById("toggle-box-checkbox").addEventListener('change', (e) => {
   if (this.checked) {
-    $('body').addClass('night')
+    $('body').classList.add("night")
   } else {
-    $('body').removeClass('night')
+    $('body').classList.remove("night")
   }
-})
+});
 
 function demo() {
   setInterval(function() {
-    $('#toggle-box-checkbox').trigger('click')
+    var event = document.createEvent('HTMLEvents');
```

Comment:
on further examination I have just noticed that this piece of code is not doing anything in our project. Now that you are here you can have the pleasure of deleting the whole thing 😁  that's the full function called demo and it's body
```js
function demo() {
  setInterval(function() {
    $('#toggle-box-checkbox').trigger('click')
  }, 1000)
}
if (document.location.pathname.indexOf('fullcpgrid') > -1) {
  demo()
}
```